lib/galaxy/datatypes/display_applications/configs/igv/vcf.xml

```diff
@@ -13,7 +13,7 @@
         <!-- We define url and params as normal, but values defined in dynamic_param are available by specified name -->
         <url>${redirect_url}</url>
         <param type="data" name="bgzip_file" url="galaxy_${DATASET_HASH}.vcf.gz" format="vcf_bgzip" allow_cors="true" mimetype="application/octet-stream"/>
-        <param type="data" name="tabix_file" metadata="tabix_index" url="galaxy_${DATASET_HASH}.vcf.gz.tbi" allow_cors="true" mimetype="application/octet-stream"/>
+        <param type="data" name="tabix_file" dataset="bgzip_file" metadata="tabix_index" url="galaxy_${DATASET_HASH}.vcf.gz.tbi" allow_cors="true" mimetype="application/octet-stream"/>
```

Comment:
This makes sense, we can skip the two-part conversion to bgzip and tabix, but this probably also hides a bug with finding the converted dataset.

Reply:
I was wondering that too and I doubt that it did work before, and I agree accessing the `metadata` of an already converted dataset is probably better than having to further convert it first.